lib/update/updater.js

```diff
@@ -181,9 +181,23 @@ Updater.prototype._update = function _update () {
 
 function noop () {}
 
+Updater.prototype.updateHeaders = function updateHeaders (options) {
+  var dataPath = path.resolve(__dirname, '..', this.config.dataPath)
+  var machineInfo = this.machineInfo.load(dataPath)
+
+  if (machineInfo.active) {
+    return _.assign(options, {
+      'device-machine-id': machineInfo.deviceId,
+      'device-machine-name': machineInfo.deviceName
+    })
```

Comment:
what's the reason behind having -machine- in the mddle? We usually call machines "devices" throughout our codebase

Reply:
httpsOptions.headers already have a property 'device-id', the -machine- is pretty much to differentiate